packages/contentstack-utilities/src/cli-ux.ts

```diff
@@ -41,15 +42,23 @@ class CLIInterface {
     this.loading = !this.loading;
   }
 
-  table(data: Array<object>, columns: any, options: object): void {
+  table(
+    data: Record<string, unknown>[],
+    columns: Table.table.Columns<Record<string, unknown>>,
+    options?: Table.table.Options
+  ): void {
     cliux.table(data, columns, options);
   }
 
   async inquire<T>(inquirePayload: InquirePayload): Promise<T> {
     inquirePayload.message = messageHandler.parse(inquirePayload.message);
     const result = await inquirer.prompt(inquirePayload as inquirer.QuestionCollection<T>);
+
     return result[inquirePayload.name] as T;
   }
 }
 
 export default new CLIInterface();
+
+export * as CliUx from '@oclif/core/lib/cli-ux'
```

Comment:
we should be exporting only CLIInterface instance

Reply:
removed other export. Using only CLIInterface